ffcx/codegeneration/ufc.h

```diff
@@ -103,19 +123,28 @@ extern "C"
         double* restrict reference_values, int order, int num_points,
         const double* restrict X);
 
+    /// Map the reference values on to the cell
+    /// @param[in] edge_reflections An array of bools to say whether or not each
+    /// each edge has been reflected. This is used to ensure that vector dofs
+    /// are correctly oriented.
+    /// @param[in] face_reflections An array of bools to say whether or not each
+    /// each face has been reflected. This is used to ensure that vector dofs
+    /// are correctly oriented.
+    /// TODO: Use std::vector<int32_t> to store 1/0 marker for each edge/face
```

Comment:
What do you mean with std::vector? FFCX and UFC is pure C.

Reply:
These comments were meant to be in dolfin (eg here: https://github.com/FEniCS/dolfinx/blob/ca87bfcf17745f301e4b664871eddae1e02f8619/cpp/dolfinx/mesh/Topology.h#L122), and were included in FFC also by mistake. I've removed them